lib/myfinance/resources/financial_account.rb

```diff
@@ -46,7 +81,7 @@ def update(id, entity_id, params = {})
       #   Documentation: https://app.myfinance.com.br/docs/api/receivable_accounts#delete_destroy
       #
       def destroy(id, entity_id)
-        http.delete(endpoint_for(id, entity_id, :destroy)) do |response|
+        http.delete(endpoint_for(id, entity_id, __method__)) do |response|
```

Comment:
Unused block argument - `response`. You can omit the argument if you don't care about it.